packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx

```diff
@@ -428,7 +348,17 @@ const CreateServiceWizard: React.FC<Props> = ({
       const templateSettings = templateMapping[templateType];
 
       if (currentProject) {
-        const plugins = createResourcePlugins(databaseType, authType);
+        const pluginIds: string[] = [databaseType];
+        if (authType && authType !== "no") {
+          pluginIds.push(authType);
```

Comment:
For .NET we will only have one auth plugin so this line can/should move to be inside the `else` section (line 358)

Reply:
Please also consider if "no" makes sense here since you removed this option in `WizardResourceSchema.ts`

Reply:
ok for the first one,
"no" is still a valid value